modules/sievefilters/modules.php

```diff
@@ -909,53 +949,51 @@ public function process() {
             }
             if ($action->action == 'redirect') {
                 $custom_condition->addAction(
-                    new \PhpSieveManager\Filters\Actions\RedirectFilterAction(['address' => $action->value])
-                );
-            }
-            if ($action->action == 'forward') {
-                $custom_condition->addAction(
-                    new \PhpSieveManager\Filters\Actions\RedirectFilterAction(['address' => $action->value])
-                );
-                $custom_condition->addAction(
-                    new \PhpSieveManager\Filters\Actions\KeepFilterAction()
+                    new \PhpSieveManager\Filters\Actions\RedirectFilterAction([$action->value])
                 );
             }
             if ($action->action == 'flag') {
                 $custom_condition->addAction(
-                    new \PhpSieveManager\Filters\Actions\FlagFilterAction(['flags' => [$action->value]])
+                    new \PhpSieveManager\Filters\Actions\FlagFilterAction([$action->value])
```

Comment:
Why do you change all these FilterAction calls to constructors? I believe php-sieve-manager still expects the params in an associative array, doesn't it?

Reply:
yes

Reply:
@kroky  means that `php-sieve-manager` expects to receive associative arrays. with the changes you have applied, it will break sieve.

Reply:
https://github.com/cypht-org/cypht/pull/1093